### ISSUE TITLE
Please remove the miner from the core wallet

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#858]
The setgenerate function is no longer functional since Ravencoin moved to the KAWPOW mining algo. Please disable it and link to the open source kawpow miner.

**Comment by Haikv** [comment]
I lost all my coins from my cold wallet after updating to this shit coin's wallet core.

**Comment by Arnauw** [comment]
> I lost all my coins from my cold wallet after updating to this shit coin's wallet core.

That's a cool story.

**Comment by H4b1ts** [comment]
Good comment @jeroz1

**Comment by lsji07** [comment]
Alternatively someone could write an extremely slow cpu kawpow miner just for testnet. That is the one place a builtin slow miner is actually useful. Save the planet etc etc and still keep the testnet blocks flowing.

**Comment by Haikv** [comment]
> 
> 
> > I lost all my coins from my cold wallet after updating to this shit coin's wallet core.
> 
> That's a cool story.

You can check online logs.

**Comment by dahug87** [comment]
Indeed, in order to improve the user experience it should be removed from the wallet, especially since some antivirus programs issue an alert which could slow down the adoption by the general public. There is also a problem with the android wallet (read comments on the playstore)
